src/hotspot/cpu/aarch64/interpreterRT_aarch64.cpp

```diff
@@ -349,7 +444,7 @@ class SlowSignatureHandler
       *_fp_identifiers |= (1ull << _num_fp_args); // mark as double
       _num_fp_args++;
     } else {
-      *_to++ = from_obj;
+      store_and_inc(_to, from_obj, NativeStack::doubleSpace);
       _num_fp_args++;
```

Comment:
`pass_byte()` and  `pass_short()` use only one `_num_int_args++;` after the `if else` but other methods use it 2 of them inside `if else`

We should be consistent.